RocketApp/Domain/Sources/RocketsClient/RocketsClient.swift

```diff
@@ -0,0 +1,26 @@
+import Combine
+import Dependencies
+import Foundation
+import Networking
+
+public struct RocketsClient {
+  public typealias GetRocketFunction = (String) -> AnyPublisher<RocketDetail, NetworkError>
+  public typealias GetAllRocketsFunction = () -> AnyPublisher<[RocketDetail], NetworkError>
+
+  public var getRocket: GetRocketFunction
+  public var getAllRockets: GetAllRocketsFunction
+}
+
+enum RocketsClientKey: DependencyKey {
```

Comment:
Zase, NetworkClient to ma v separatnich souborech a tady je to zase jinak. Musi to bejt vsude stejne...

Reply:
Dependency values a keys slouceny do Client+dependencyManaging.